### PR TITLE
Email sender

### PR DESCRIPTION
**Pull request by Filari02**
Created an email sender which informs users when item (which users added in their favorite item list) price has dropped. Do you have any ideas how to improve?

### CODE REVIEW

**Review by Tomas-Per — COMMENTED** [review]
Have you already tested if the API and the modified controller and repository works in this configuration?

File: EmailSender/EmailSettings.settings [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <Value Profile="(Default)">Libra123</Value>
+    </Setting>
+    <Setting Name="host" Type="System.String" Scope="Application">
+      <Value Profile="(Default)">smtp.gmail.com</Value>
+    </Setting>
+    <Setting Name="port" Type="System.Int32" Scope="Application">
+      <Value Profile="(Default)">587</Value>
+    </Setting>
+    <Setting Name="ssl" Type="System.Boolean" Scope="Application">
+      <Value Profile="(Default)">True</Value>
```
Comment (Tomas-Per) [inline EmailSender/EmailSettings.settings:18]: This info should be hidden

Comment (Filari02) [reply EmailSender/EmailSettings.settings:18]: done

**Comment by Filari02** [comment]
could not test api since had many erros while updating DB manually (when swagger services was offline), so didnt have valid data for testing. BUT controller was written in the same way as all other controllers so it should work

**Review by Tomas-Per — APPROVED** [review]

**Review by Ugniusst — APPROVED** [review]

**Comment by Filari02** [comment]
controllers and repository has been tested. Everything works fine now

**Pull request merged**